client/flutter/lib/pages/main_pages/check_up_intro_page.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/widgets.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:who_app/components/page_button.dart';
+import 'package:who_app/components/page_scaffold/page_header.dart';
+import 'package:who_app/components/page_scaffold/page_scaffold.dart';
+import 'package:who_app/components/themed_text.dart';
+import 'package:who_app/constants.dart';
+
+class CheckUpIntroPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return PageScaffold(
+      disableBackButton: true,
+      headingBorderColor: Color(0x0),
+      color: CupertinoColors.white,
+      title: "Check-Up",
```

Comment:
nit/optional: add TODOs to localize across the entire file

Reply:
Added